util/venv-dependencies.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+set -e
+
+# quiet=--quiet
+quiet=
+
+#### Options end here.
+
+willexit=
+if [ "$#" -ne 1 ]; then
+   echo "
+Usage: $0 new-directory
+
+This script installs Streisand builder dependencies. It depends on
+a working Python 2.7 'pip' command on your PATH. I'll install 
+virtualenv for you, but on Linux, this requires sudo access.
+
+'new-directory' must be somewhere you can write to. I suggest
+$HOME/streisand-deps. 
+"
+   willexit=1
+fi
+
+if ! pip >/dev/null; then
+   echo "
+You need a working 'pip' command. To get one:
+
+On Ubuntu and WSL:
+   sudo apt-get install python-pip
+
+On macOS:
+   # If you haven't, install homebrew from https://brew.sh/
+   brew install python
+"
+   willexit=1
+fi
+
+if [ -n "$willexit" ]; then
+    exit 1
+fi
+
+die () {
+    echo "$@"
+    exit 1
+}
+
+sudo_pip () {
+    # pip complains loudly about directory permissions when sudo without -H.
+    sudo -H pip $quiet "$@"
+}
+
+our_pip () {
+    pip $quiet "$@"
+}
+
+our_pip_install () {
+    our_pip install "$@"
+}
+
+# An easy way to see if Homebrew is installed.
+if brew command command >/dev/null; then
```

Comment:
I think you need to also redirect stderr here. Running this script from a Ubuntu instance produced:
> util/venv-dependencies.sh: line 61: brew: command not found